### DISCUSSION TITLE
关于access_token多项目共享

### DISCUSSION THREAD

**Discussion by tngtion** [w7corp/easywechat#2497] [Q&A]
因为多个项目用到同一个公众号。。但是access_token刷新会冲突。。已经做了中控刷新access_token了。怎么样才可以让easywechat不自动去刷新access_token

或者哪位大神可以说一下easywechat是在什么时候刷新access_token的

我可以自定义吗？
目前我发现。access_token是会自己刷新的。
看了源码。没有找到刷新的地方。

有地方说是初始化传入config的时候就刷新了。

求攻略。怎么多项目情况下维护access_token

**Comment by zbuzhi92** [comment]
可以在实例化的时候 公用一个缓存驱动，比如redis、memcache...
